src/main/resources/assets/skyhanni/emoji/emojis

```diff
@@ -0,0 +1 @@
+emoji_table.png is a modified version of https://github.com/jdecked/twemoji
```

Comment:
pretty sure you legally have to give proper attribution, unsure if this counts. https://github.com/jdecked/twemoji#attribution-requirements